src/amr/data/particles/refine/split_2d.h

```diff
@@ -0,0 +1,481 @@
+/*
+Splitting reference material can be found @
+  https://github.com/PHAREHUB/PHARE/wiki/SplitPattern
+*/
+
+#ifndef PHARE_SPLIT_2D_H
+#define PHARE_SPLIT_2D_H
+
+namespace PHARE::amr
+{
+using namespace PHARE::core;
+
+template<>
+struct ExactSplitParticleDispatcher<DimConst<2>>
+{
+    template<typename Deltas, typename DeltaVals>
+    static void set_deltas(Deltas& deltas, DeltaVals const& delta_vals, size_t const root)
+    {
+        for (size_t y = 0; y < root; y++)
+        {
+            size_t yoff = y * root;
+            for (size_t x = 0; x < root; x++)
+            {
+                deltas[x + yoff] = {delta_vals[x], delta_vals[y]};
+            }
+        }
+    }
+};
+
+
+/**************************************************************************/
+template<>
+struct SplitParticleDispatcher<DimConst<2>, RefinedParticlesConst<4>>
+{
```

Comment:
this struct does not depend on the interp order : below, the pattern is the `PURPLE` one. But for `N=5`, the pattern will be `BLACK` + `PURPLE` for interp 1, and `BLACK` + `PINK` for interp 2 & 3.